cassava_trait.obo

```diff
@@ -13908,6 +14784,13 @@ namespace: CassavaScale
 is_a: CO_334:0001001 ! Scales
 relationship: scale_of CO_334:0013051 ! white color appearance method
 
+[Term]
+id: CO_334:0103077
+name: Block:3
```

Comment:
what is 'block 3'? 
Do we want to use 'tricot block 3' for all the new variables? Are those tricot specific or should those be allowed to be reused by other projects/breeders?

Reply:
The tricot variables have all the same method and scale. This block 3 means that for each variable you get an array of 3 varieties (the best of the three , the one in the middle and the worse of the 3).